src/Acuminator/Acuminator.Analyzers/StaticAnalysis/SavingChanges/SaveOperationKind.cs

```diff
@@ -25,4 +25,29 @@ enum SaveOperationKind
 		/// </summary>
 		CachePersist,
 	}
+
+	enum PXDatabaseKind
```

Comment:
please use explicit access modifiers. Also the name PXDatabaseKind is misleading. It doesn't reflect that this enum represents db operation. I would rather guess that it represent different databases like MSSQL and MySql. Please rename to something like DBOperation or PXDBOperation or something that will reflect that this is a type of db operation

Reply:
It also won't hurt to have each class in a separate file but I won't insist on it for enums